tools/targets/__init__.py

```diff
@@ -48,7 +48,9 @@
     "Cortex-M23": ["M23", "CORTEX_M", "LIKE_CORTEX_M23", "CORTEX"],
     "Cortex-M23-NS": ["M23", "M23_NS", "CORTEX_M", "LIKE_CORTEX_M23", "CORTEX"],
     "Cortex-M33": ["M33", "CORTEX_M", "LIKE_CORTEX_M33", "CORTEX"],
-    "Cortex-M33-NS": ["M33", "M33_NS", "CORTEX_M", "LIKE_CORTEX_M33", "CORTEX"]
+    "Cortex-M33-NS": ["M33", "M33_NS", "CORTEX_M", "LIKE_CORTEX_M33", "CORTEX"],
+    "Cortex-M33F": ["M33", "CORTEX_M", "LIKE_CORTEX_M33", "CORTEX"],
```

Comment:
M33F?

Floating point?

Reply:
Saw it there :
https://github.com/ARMmbed/mbed-os/blob/c37dc5fea8107e00dcc8a7021af487f6bd858f59/tools/toolchains/__init__.py#L77

Reply:
I am not sure if we need floating point in secure mode? Also need to check can we enable floating point in non-secure only

Reply:
> I am not sure if we need floating point in secure mode? Also need to check can we enable floating point in non-secure only

@deepikabhavnani Is this change correct ?

Reply:
Yes